torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -0,0 +1,259 @@
+#include <torch/csrc/jit/codegen/cuda/scheduler.h>
+
+#include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
+#include <torch/csrc/jit/codegen/cuda/ir_iostream.h>
+#include <torch/csrc/jit/codegen/cuda/parser.h>
+#include <torch/csrc/jit/codegen/cuda/arith.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+constexpr int kUnrollFactor = 4;
+
+namespace {
```

Comment:
please use static functions instead of anonymous namespaces

Reply:
never mind, based on the conversation today, this is not the style guideline

Reply:
Per recent discussion, anonymous namespaces are the default now.